tools/github-team-user-store/GitHubTeamUserStore/GitHubTeamUserStore/TeamUserGenerator.cs

```diff
@@ -15,6 +16,10 @@ public class TeamUserGenerator
         // that this is the list of Logins (Octokit.User.Login) which are what's used in CODEOWNERS, @mentions, etc.
         private static Dictionary<string, List<string>> _teamUserDict = new Dictionary<string, List<string>>();
 
+        // The user list of azure-sdk-write will contain a distinct list of all users from all child teams. For
+        // each one, check if they're a member of azure and store that in here.
+        private static Dictionary<string, bool> _userOrgDict = new Dictionary<string, bool>();
```

Comment:
Similar feedback about having unnecessary static global values (similar to the teamUserDic which I would have similar feedback on). While what you have will work it is generally a bad pattern if this library did get used by other things.

Reply:
This will be updated with the next commit.